test/flint/fq-test.jl

```diff
@@ -1,35 +1,23 @@
-function test_elem(R::FqPolyRepField)
-  return rand(R)
-end
-
 @testset "FqPolyRepFieldElem.conformance_tests" begin
-  test_Field_interface_recursive(Native.finite_field(ZZRingElem(7), 5, "z")[1])
-
-  Sy, y = polynomial_ring(residue_ring(ZZ, 36893488147419103363)[1], "y")
-  T, z = Native.finite_field(y^2 + 1, "z")
-  test_Field_interface_recursive(T)
-
-  Syy, yy = polynomial_ring(Native.GF(ZZRingElem(36893488147419103363)), "y")
-  T2, z2 = Native.finite_field(yy^2 + 1, "z")
-  test_Field_interface_recursive(T2)
```

Comment:
why did you remove the above tests?

Reply:
I removed them because they seemed to be redundant: they all test a `FqPolyRepField`, just constructed in different ways. But we already test the constructors elsewhere, and for testing the ring conformance, it shouldn't matter how we constructed the field.